.travis.yml

```diff
@@ -6,11 +6,17 @@ node_js:
 
 matrix:
   fast_finish: true
+  include:
+    - node_js: 7 # Version used to deploy to npm registry
+      env: BUILD_ENV=production
 
 cache:
   directories:
     - ~/.npm
 
+before_script:
+  - NODE_ENV=$BUILD_ENV npm run build
+
```

Comment:
This works fine when `BUILD_ENV` is not defined, which is the case outside of that `include`d combination.